### PR TITLE
WL-757 Placed correct icon for 'Sites' tool.

### PR DESCRIPTION
**Pull request by ouit0408**
Edited the portal.css for correct placement and alignment of site icon.

### CODE REVIEW

File: library/src/webapp/skin/neo-ox/portal.css [hunk 1]
```diff
@@ -1448,7 +1448,9 @@ ul#otherSiteList li a span, ul.otherSitesCategorList li a span, ul.otherQuickLin
     background-position: -162px -252px;
 }
 .icon-sakai-sitebrowser {
-    background-position: -468px -522px;
+    background-position: center;
+    background-repeat: initial;
+    background-image: url(/library/image/silk/page_white_find.png) !important
```
Comment (buckett) [inline library/src/webapp/skin/neo-ox/portal.css]: Could we move this out of the silk icon into the skin folder as well, as it shouldn't have ever been in there in the first place. Also I don't think you need the `!important`

Comment (ouit0408) [inline library/src/webapp/skin/neo-ox/portal.css]: Which path for the icon  do you suggest?

Comment (buckett) [inline library/src/webapp/skin/neo-ox/portal.css]: Inside the skin folder for neo-ox again.

Comment (ouit0408) [inline library/src/webapp/skin/neo-ox/portal.css]: Placed the icon in the skin of neo-ox folder and have pushed the code.

**Pull request merged**